.travis.yml

```diff
@@ -1,10 +1,9 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.2.5
-  - 2.3.1
   - 2.5.3
-before_install: gem install bundler -v 1.12.5
+  - 2.6.0
```

Comment:
Should I add another Ruby version? Considering that Ruby 2.4 is supported till april of 2020.